dev/com.ibm.ws.security.authentication.filter/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -30,6 +30,8 @@ name.desc=Specifies the name.
 
 value=Value
 value.desc=Specifies the value.
+valueOptional.desc=Specifies the value. If value is not specify, matching is done with the key not value.
```

Comment:
Hi @utle - Could you help me fill in a few blanks here? I'm thinking this will end up something like:

"The value **<is this an attribute, etc.?>** specifies the value of **<what value does this specify?>**. If the value is not specified, then **<what is matched with the key?>** is matched with the key. "

What would be the correct information to insert for the bolded items?

Reply:
This is the HTTP request header name and value. For example email=myemail@xyz.com... so we don't really know the attribute name or value. Before this change, the value is required but now optional so I create a new valueOtional.desc.

Reply:
@utle Agreed on:
"The value attribute specifies the HTTP request header value. If the value is not specified, then matching is done with the name attribute, not the value attribute."
Thanks!

Reply:
Done.